ui/app/AppLayouts/Onboarding2/OnboardingFlow.qml

```diff
@@ -197,8 +197,8 @@ SQUtils.QObject {
 
         onFinished: (fromBackupSeedphrase) => {
             d.flow = fromBackupSeedphrase
-                        ? Onboarding.SecondaryFlow.CreateProfileWithKeycardExistingSeedphrase
-                        : Onboarding.SecondaryFlow.CreateProfileWithKeycardNewSeedphrase
+                        ? Onboarding.SecondaryFlow.CreateProfileWithKeycardNewSeedphrase
```

Comment:
This was swapped but luckily in both places so it worked correctly; it's just confusing

Reply:
from backkup -> existing

probably it was ok, matter of interpretation probably, some wording alignments needed probably